helix-core/src/main/java/org/apache/helix/controller/changedetector/ResourceChangeSnapshot.java

```diff
@@ -82,10 +82,10 @@ class ResourceChangeSnapshot {
     _changedTypes = new HashSet<>(dataProvider.getRefreshedChangeTypes());
 
     _instanceConfigMap = ignoreNonTopologyChange ?
```

Comment:
Do we want to change the name of `_instanceConfigMap ` as well? If user call `getInstanceConfigMap` it may be confusing.